hana/cds-plugin.js

```diff
@@ -1,6 +1,9 @@
 const cds = require('@sap/cds/lib')
 
+if (!cds.env.fiori.lean_draft) {
+  throw new Error('"@cap-js/hana" only works if cds.fiori.lean_draft is enabled. Please adapt your configuration.')
+}
+
```

Comment:
Why do we need to set `cds.env.sql.dialect = 'hana'`?

Reply:
Then this is basically "only" required for our tests which generate the plain "CREATE" statements as of now. In production, the deployment is anyway done by HDI?

Reply:
Yes, but quoted mode is not yet supported, is it?